src/v3/src/mocks/response/idp/idx/introspect/authenticator-enroll-grace-period-with-skip.json

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
Can probably remove the mock file here and everything under v3/src/mocks if preferred. I was previously using it with msw for local testing